yarn-project/end-to-end/src/e2e_ordering.test.ts

```diff
@@ -17,7 +17,9 @@ describe('e2e_ordering', () => {
   const expectLogsFromLastBlockToBe = async (logMessages: bigint[]) => {
     const l2BlockNum = await pxe.getBlockNumber();
     const unencryptedLogs = await pxe.getUnencryptedLogs(l2BlockNum, 1);
-    const unrolledLogs = L2BlockL2Logs.unrollLogs(unencryptedLogs);
+    const unrolledLogs = L2BlockL2Logs.unrollLogs(unencryptedLogs)
```

Comment:
I will make this nicer in [this PR](https://github.com/AztecProtocol/aztec-packages/pull/2600).